packages/service/lib/schema.js

```diff
@@ -600,6 +600,10 @@ const openApiBase = {
     prefix: {
       type: 'string',
       description: 'Base URL for the OpenAPI'
+    },
+    file: {
+      type: 'string',
```

Comment:
please use our keyword to resolve relative paths.

Also, please rename this to path